packages/es-dev-server/package.json

```diff
@@ -78,6 +80,7 @@
     "babel-preset-typescript": "^7.0.0-alpha.19",
     "chai": "^4.2.0",
     "koa-proxies": "^0.8.1",
+    "lit-element": "2.0.0",
```

Comment:
any particular reason we have pinned versions here?

Reply:
Yes a very silly reason that I want to make sure it ends up in the node_modules of es-dev-server, so that they're within the scope of the web server.

I know that's pretty fragile, but it's just for the demo of the dev server.

An alternative can be to always serve from the root of the monorepo.

Reply:
hihi ok... good enough I would say 😁